wd_demo.au3

```diff
@@ -1263,3 +1264,47 @@ Func SetupOpera($bHeadless)
 	Local $sCapabilities = _WD_CapabilitiesGet()
 	Return $sCapabilities
 EndFunc   ;==>SetupOpera
+
+Func SetupEdgeIEMode() ; this is for MS Edge IE Mode
+	#REMARK In order to properly use MS Edge In IE Mode you need To Do some settings - Set the "Internet, Local intranet, Trusted sites, Restricted sites" to protected mode, enabling or disabling it is not a matter
+	; https://www.edureka.co/community/1690/problem-using-internetexplorerdriver-selenium-webdriver
+	Local $sTimeStamp = @YEAR & '-' & @MON & '-' & @MDAY & '_' & @HOUR & @MIN & @SEC
+	; https://www.selenium.dev/documentation/ie_driver_server/#required-configuration
+	_WD_Option('Driver', 'IEDriverServer.exe') ;
+	_WD_Option('Port', 5555)
+	_WD_Option('DriverParams', '--verbose --log-path="' & @ScriptDir & '\' & $sTimeStamp & ' EdgeIEMode.log" -log-level=INFO' & " --port=" & $_WD_PORT & " -host=127.0.0.1" )
+#cs
+	Local $sCapabilities = '{"capabilities": {"alwaysMatch": { "se:ieOptions" : { "ie.edgepath":"C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe", "ie.edgechromium":true, "ignoreProtectedModeSettings":true,"excludeSwitches": ["enable-automation"]}}}}'
+	Local $sCapabilities = _
+			'{' & @CRLF & _
+			'    "capabilities": {' & @CRLF & _
+			'        "alwaysMatch": {' & @CRLF & _
+			'            "se:ieOptions": {' & @CRLF & _
+			'                "ie.edgepath": "C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe",' & @CRLF & _
+			'                "ie.edgechromium":true,' & @CRLF & _
+			'                "ignoreProtectedModeSettings":true,' & @CRLF & _
+			'                "excludeSwitches": ["enable-automation"]' & @CRLF & _
+			'            }' & @CRLF & _
+			'        }' & @CRLF & _
+			'    }' & @CRLF & _
+			'}' & @CRLF & _
+			''
+	ConsoleWrite("! $sCapabilities (by hand) = " & @CRLF & $sCapabilities & @CRLF)
+#ce
+
+	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgepath")
+	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgechromium")
+	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ignoreProtectedModeSettings")
```

Comment:
If these are required for IEMode to work correctly, then they should be included by default.

Reply:
IMO, they should already be defined so that the end user should be forced to add these via `_WD_CapabilitiesDefine`.

Reply:
after removig:
```
_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgepath")
```

all was fine, but after removing:
```autoit
_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgechromium")
```
I starts getting issues:
![image](https://user-images.githubusercontent.com/11089482/233435830-b9764bf7-2688-4ab6-a99b-489d9103ff98.png)

Reply:
using:
```autoit
	_WD_CapabilitiesAdd('alwaysMatch', 'msedgeie')
	_WD_CapabilitiesAdd('w3c', True)
	_WD_CapabilitiesAdd("ie.edgechromium", True)
;~ 	_WD_CapabilitiesAdd("ignoreProtectedModeSettings", True)
	_WD_CapabilitiesAdd("initialBrowserUrl", "https://google.com")
	_WD_CapabilitiesAdd('excludeSwitches', 'enable-automation')
```
I get:
```
! _WD_LastHTTPResponse = {
	"value" : 
	{
		"error" : "session not created",
		"message" : "Unexpected error launching Internet Explorer. Protected Mode settings are not the same for all zones. Enable Protected Mode must be set to the same value (enabled or disabled) for all zones.",
		"stacktrace" : ""
	}
}
```

Reply:
so finally: [ie.edgechromium - not needed](https://github.com/Danp2/au3WebDriver/pull/454/commits/e0e25c25fa5b459fe1b6d28c8c3a86061e99be2e)

Reply:
Ok, but that doesn't satisfy my objection

> they should already be defined so that the end user isn't forced to add these via _WD_CapabilitiesDefine.

Do you still not understand what I am wanting here?

Reply:
Do you mean that instead using:
```autoit
	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgepath")
	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ie.edgechromium")
	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "ignoreProtectedModeSettings")
	_WD_CapabilitiesDefine($_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, "initialBrowserUrl")
```
I shoud change wd_capabilities.au3
```autoit
Global $_WD_KEYS__SPECIFICVENDOR_PRIMITIVE = _
		'\A(binary|debuggerAddress|detach|minidumpPath|w3c|ie.edgepath|ie.edgechromium|ignoreProtectedModeSettings|initialBrowserUrl)\Z'
```

Reply:
So take a look on:
[demo: _WD_CapabilitiesDefine - removed](https://github.com/Danp2/au3WebDriver/pull/454/commits/071067ff65b303693a0eaa5d376a54c7e0af4c74)

and
[Global $_WD_KEYS__SPECIFICVENDOR_PRIMITIVE](https://github.com/Danp2/au3WebDriver/pull/454/commits/d793353acafbfcf3c105a814cf8fedce2b0c9b6a)